client/client_test.go

```diff
@@ -81,7 +83,7 @@ func newNotifyingSender(wrapped client.KVSender) *notifyingSender {
 // an HTTP sender to the server at addr.
 // It contains a waitgroup to allow waiting.
 func createTestNotifyClient(addr string) *client.KV {
-	sender, err := client.NewHTTPSender(addr, testutils.NewTestBaseContext())
+	sender, err := http.NewHTTPSender(addr, testutils.NewTestBaseContext())
```

Comment:
Go encourages avoiding the repetition of http here: `s/http.NewHTTPSender/http.NewSender/g`. On the other hand, this is likely going away soon and `NewHTTPSender` will be unexported. Not sure what your interests are after this PR, but we could use some help moving from `client.KV` to `client.DB`. See #997.